root/components/relationship-editor.tt

```diff
@@ -10,11 +10,11 @@
 </div>
 
 <script type="text/html" id="template.relationship-dialog">
-  <!-- ko if: relationship().editsPending -->
+  <!-- ko if: $data.editing && $data.editing.editsPending -->
```

Comment:
The comments are Knockout, they call it a virtual element or something (it’s for grouping several tags without forcing us to introduce an unnecessary `<div>`, like React does it).